manifests/install/windows.pp

```diff
@@ -8,33 +8,49 @@
 
   remote_file {"${filebeat::tmp_dir}/${filename}.zip":
     ensure      => present,
-    source      => $filebeat::download_url,
+    source      => $filebeat::real_download_url,
     verify_peer => false,
     proxy       => $filebeat::proxy_address,
   }
 
   exec { "unzip ${filename}":
     command  => "\$sh=New-Object -COM Shell.Application;\$sh.namespace((Convert-Path '${filebeat::install_dir}')).Copyhere(\$sh.namespace((Convert-Path '${filebeat::tmp_dir}/${filename}.zip')).items(), 16)",
-    creates  => "${filebeat::install_dir}/Filebeat",
+    creates  => "${filebeat::install_dir}/Filebeat/${filename}",
     provider => powershell,
     require  => [
       File[$filebeat::install_dir],
       Remote_file["${filebeat::tmp_dir}/${filename}.zip"],
     ],
   }
 
-  exec { 'rename folder':
-    command  => "Rename-Item '${filebeat::install_dir}/${filename}' Filebeat",
-    creates  => "${filebeat::install_dir}/Filebeat",
+  # You can't remove the old dir while the service has files locked...
+  exec { "stop service ${filename}":
+    command  => "Set-Service -Name filebeat -Status Stopped",
```

Comment:
Does this need to be in double quotes? If it does I can add a [control comment](http://puppet-lint.com/controlcomments/), but I don't want to risk breaking something since I really know nothing about running this in Windows. I don't mind making the change, but don't want to break something without a Windows test environment set up.